docker-compose.connect.yml

```diff
@@ -32,6 +32,14 @@ services:
             - traefik.http.routers.deviceconnectMgmt.tls=true
             - traefik.http.routers.deviceconnectMgmt.service=deviceconnectMgmt
             - traefik.http.services.deviceconnectMgmt.loadbalancer.server.port=8080
+
+            - traefik.http.routers.deviceconnectMgmtDownload.entrypoints=https
+            - traefik.http.routers.deviceconnectMgmtDownload.rule=PathPrefix(`/api/management/{(v[0-9]+)}/deviceconnect/devices/{[0-9a-zA-Z\-]+}/download`)
+            - traefik.http.routers.deviceconnectMgmtDownload.middlewares=userauth,sec-headers,compression,json-error-responder1,json-error-responder3,json-error-responder4
+            - traefik.http.routers.deviceconnectMgmtDownload.tls=true
+            - traefik.http.routers.deviceconnectMgmtDownload.service=deviceconnectMgmt
+            - traefik.http.services.deviceconnectMgmtDownload.loadbalancer.server.port=8080
+ 
```

Comment:
I know why you are doing it (otherwise Traefik masks the 5xx), but I believe we should do it that way.
As long as deviceconnect doesn't answer with 5xx without a JSON payload anywhere, we can remove the json-error-responder2 from the main router instead of creating a new one.

Reply:
I am more sure that /download responds with valid 500 json than about whole deviceconnect. thats the reason, does it make sense?

Reply:
@merlin-northern yes, I checked and all the InternalServerError from deviceconnect correctly return a JSON with the "error" key.

Reply:
now you are telling me? pushed it hours ago :)

Reply:
it doesn't matter